tests/utils/test_serialization.py

```diff
@@ -29,4 +29,8 @@ def test_list():
 
 def test_load():
     with pytest.raises(ValueError):
-        serialization.load_state_dict("path", "fsdp", "tp")
+        serialization.load_state_dict(
```

Comment:
is this our only test case for the function? maybe we can add more?

Reply:
changed this tests to be more thorough